Helper_Scripts/checks/email_search_dual_read_parity.py

```
@@ -51,20 +51,18 @@
 
 ensure_repo_root()
 
-if TYPE_CHECKING:
-    from tldw_Server_API.app.core.DB_Management.Media_DB_v2 import MediaDatabase
+try:
+    from tldw_Server_API.app.core.DB_Management.media_db.api import create_media_database
+except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)